libs/application-generic/src/usecases/get-subscriber-global-preference/get-subscriber-global-preference.usecase.ts

```diff
@@ -111,7 +110,9 @@ export class GetSubscriberGlobalPreference {
     );
 
     if (!subscriber) {
-      throw new ApiException(`Subscriber ${command.subscriberId} not found`);
+      throw new NotFoundException(
+        `Subscriber ${command.subscriberId} not found`,
+      );
```

Comment:
did you verify that this usecase is not used in worker?

Reply:
Yes - from what I see, worker is using just `GetSubscriberPreference` usecase, but even for that usecase I can't find any use inside worker - I will check if we can remove it possibly in next PR.